fdbclient/include/fdbclient/BlobCipher.h

```diff
@@ -508,7 +537,7 @@ struct BlobCipherEncryptHeaderRef {
 
 #pragma pack(push, 1) // exact fit - no padding
 typedef struct BlobCipherEncryptHeader {
-	static constexpr int headerSize = 104;
+	static constexpr int headerSize = 112;
```

Comment:
Do we need to version this header? If we decide to deprecate this in favor of Configurable encryption then it should be fine but something to consider

Reply:
I think given we plan to keep the old-format for now, it might be a good idea to add one. 
However, it is better done as a separate PR, for now the feature is not enabled by default (also _not_ declared the feature available as of today), hence, it is OK to change the on-disk format.